src/olympia/accounts/views.py

```diff
@@ -330,7 +325,7 @@ def get(self, request, user, identity, next_path):
 
 def logout_user(request, response):
     logout(request)
-    remove_api_token_cookie(response)
+    response.delete_cookie(API_TOKEN_COOKIE)
```

Comment:
Sorry but either I'm blind or I don't get it, didn't you just move the code from `remove_api_token_cookie` to the `logout_user` function? How does this solve the issue?

Reply:
Sorry, it was late and I messed up :) I had the `domain=settings.DOMAIN` in there but changed it to see if I could make a test fail (I couldn't, per django ticket linked). I'll add it back.